### PR TITLE
Stardock Fences v 4.11.5 update

### PR DESCRIPTION
**Pull request by blckpythn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109446&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: 4.11.5
```
Comment (ItzLevvie) [inline manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.installer.yaml:5]:
```suggestion
PackageVersion: 4.1.1.5
```
This must match Control Panel.

You also have to change `manifests/s/Stardock/Fences/4.11.5` to `manifests/s/Stardock/Fences/4.1.1.5`

File: manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: 4.11.5
```
Comment (ItzLevvie) [inline manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.1.1.5
```
This must match Control Panel.

You also have to change `manifests/s/Stardock/Fences/4.11.5` to `manifests/s/Stardock/Fences/4.1.1.5`

File: manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Stardock.Fences
+PackageVersion: 4.11.5
```
Comment (ItzLevvie) [inline manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.yaml:5]:
```suggestion
PackageVersion: 4.1.1.5
```
This must match Control Panel.

You also have to change `manifests/s/Stardock/Fences/4.11.5` to `manifests/s/Stardock/Fences/4.1.1.5`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.locale.en-US.yaml (hunk 2 above)
Comment (blckpythn) [reply manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.locale.en-US.yaml]: Updated to 4.1.1.5 for path and PackageVersion

File: manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.yaml (hunk 3 above)
Comment (blckpythn) [reply manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.yaml]: Updated to 4.1.1.5 for path and PackageVersion

File: manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.installer.yaml (hunk 1 above)
Comment (blckpythn) [reply manifests/s/Stardock/Fences/4.11.5/Stardock.Fences.installer.yaml]: Updated to 4.1.1.5 for path and PackageVersion

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**